app/Models/Asset.php

```diff
@@ -72,8 +72,7 @@ public function declinedCheckout(User $declinedBy, $signature)
 
     protected $casts = [
         'purchase_date' => 'date',
-        'asset_eol_date' => 'date',
```

Comment:
This _feels_ like the wrong solution - because it *is* a date, and it's a date in the database, and it makes sense to cast it as such. But, if this makes the problem go away, I'll allow it.

Reply:
Yeah, but remember how we had to pull all those casts a while back because date validation was going weird? I think we just never pulled that one.